nist/int.go

```diff
@@ -346,6 +343,18 @@ func (i *Int) BigEndian(min, max int) []byte {
 	return buf
 }
 
+// Set value to a number represented by a byte string.
+// Endianness depends on the endianess set in i.
+func (i *Int) SetBytes(a []byte) abstract.Secret {
+	var buff = a
+	if i.Bo == LittleEndian {
+		buff = make([]byte, len(a))
+		util.Reverse(buff, a)
```

Comment:
I just realized that util.Reverse() should really have append()-like or Hash()-like semantics, allocating or extending the destination slice automatically if it is too short.  That simplify this and probably other calling code - e.g., here, this would become simply 'a = util.Reverse(nil, a)', and you could delete the 'buff' variable entirely.